### PR TITLE
updated Project 1 with nouns, actions and pseudo code functions

### PR DESCRIPTION
**Pull request by metaphysician**
hi matkatmusic, here's the pull request for Project 1 part 1

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +74,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  action 3:bear rubs its tummy
+bear.rubTummy(); 
+//  7) 
+//  Noun:lawnmower   
+//  action 1:rotates blade
+lawnmower.rotateBlade();
+//  action 2:stops blade rotation
+lawnmower.stopBladeRotation();
+//  action 3:sets lawnmower in gear so it will move
+lawnmower.engageDrivetrain();
```
Comment (matkatmusic) [inline main.cpp]: `lawn mower` is 2 words in the english language.  use camelCase to name it properly
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -52,85 +74,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  Noun:phone   
 //  action 1:
-
+phone.connectCall();
 //  action 2:
-
+phone.updateOS();
 //  action 3:
- 
+phone.applySettings();
```
Comment (matkatmusic) [inline main.cpp]: Where are your plain-english actions? 
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -52,85 +74,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  action 3:ball stops moving
+ball.stop()
+//  5) 
+//  Noun:game
+//  action 1:game starts
+game.start();
+//  action 2:game pauses playback
+game.pauses();
+//  action 3:game finishes
+game.finish();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (metaphysician) [reply main.cpp]: should be corrected now

File: main.cpp (hunk 2 above)
Comment (metaphysician) [reply main.cpp]: added plain english actions for phone

File: main.cpp (hunk 3 above)
Comment (metaphysician) [reply main.cpp]: replaced 'game' with 'hardDisk' and changed actions and English descriptions

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**